### ISSUE TITLE
Test failure with python 3.10a4

### ISSUE THREAD

**Issue by opoplawski** [enthought/pyface#873]
Doing early test in Fedora against python 3.10a.  Tests tail with:
```
test_bind (pyface.tests.test_python_shell.TestPythonShell) ... Traceback (most recent call last):
  File "/builddir/build/BUILD/pyface-7.2.0/pyface/ui/qt4/console/console_widget.py", line 1969, in _adjust_scrollbars
    scrollbar.setPageStep(step)
TypeError: setPageStep(self, int): argument 1 has unexpected type 'float'
```
https://github.com/enthought/pyface/blob/master/pyface/ui/qt4/console/console_widget.py#L1969

Perhaps need to cast to int after the division

**Comment by rahulporuri** [comment]
Thanks for the report @opoplawski . I'm not sure why this is reported only on Python 3.10a4. I'm not sure what changed between 3.9 and 3.10 that raises the error.

**Comment by rahulporuri** [comment]
closing as this got fixed in #958
